qgreenland/constants.py

```diff
@@ -42,7 +42,7 @@
 # renaming. Make it a subdir?
 TMP_DIR = WIP_DIR
 
-REQUEST_TIMEOUT = 20
+REQUEST_TIMEOUT = 30
```

Comment:
This is in seconds right? While we're looking at this, a comment would be nice to remind ourselves of that.